### DISCUSSION TITLE
v0.2.2-beta.0

### DISCUSSION THREAD

**Discussion by daffamumtaz2361** [cstayyab/WALC#163] [General]
Hello, long time no see. @cstayyab has transferred ownership to WAClient organization. No time for long speech right now, so let's get going with the update. There is not much change in this update, I just updated all the dependency to make sure the app works. Because there's just so many libraries to update, I need your help to test it to make sure it's bug free when it releases.

<hr /><em>This discussion was created from the release <a href='https://github.com/WAClient/WALC/releases/tag/v0.2.2-beta.0'>v0.2.2-beta.0</a>.</em>

**Comment by biase-d** [comment]
![image](https://user-images.githubusercontent.com/47496233/133207951-35a57632-9294-47e6-9cb9-7e013fff82ec.png)
I have been getting this recently

**Reply by pacnica** [reply]
Exactly same image.

**Reply by daffamumtaz2361** [reply]
First of all make sure you're not using whatsapp web beta and try logging out from your phone.

**Reply by pacnica** [reply]
I'm using Walc.Appimage and closed all session and restart... Now, all right!

**Reply by Hexcz** [reply]
A workaround for that is opening a second stance while first is still open. Waiting for a actual fix tho.
